.travis.yml

```diff
@@ -7,6 +7,7 @@ before_install:
 install:
   - ./travis-tool.sh r_binary_install knitr
   - ./travis-tool.sh r_binary_install testthat
+  - ./travis-tool.sh r_install pryr
```

Comment:
I was thinking about inlining, too, but that would need compilation for magrittr, which is not good, either. I agree that having `pryr` in `Suggests` is the best compromise. Maybe I can suggest R-core to add these two functions to base R. :)